WebUI/src/routes/+page.svelte

```diff
@@ -36,12 +44,17 @@
 <div class="flex flex-col items-center justify-center h-full">
   <div class="flex-col space-y-5 w-full max-w-md">
     <WiFiList />
+
     <div class="flex flex-col space-y-2">
-      <h3 class="h3">Other settings</h3>
+      <h3 class="h3">Gateway Pairing</h3>
```

Comment:
"gateway pairing"? u mean the regular pairing?

Reply:
Reverting this.